Cargo.toml

```diff
@@ -7,7 +7,7 @@ authors = ["Stratis Developers <stratis-devel@lists.fedorahosted.com>"]
 dbus = {version = "0.6.4", optional = true}
 clap = "2"
 nix = "0.13"
-devicemapper = "0.26.0"
+devicemapper = {git="https://github.com/mulkieran/devicemapper-rs", branch="master-error"} 
```

Comment:
don't forget to remove this

Reply:
New version of devicemapper-rs is available, so you can bump the version for this PR.